libs/arangodb/langchain_arangodb/graphs/arangodb_graph.py

```diff
@@ -321,10 +321,24 @@ def generate_schema(
             if collection["system"] or collection["name"] not in collection_names:
                 continue
 
-            # Extract collection name, type, and size
+            # Extract collection name, type, size, and indexes
             col_name: str = collection["name"]
             col_type: str = collection["type"]
             col_size: int = self.db.collection(col_name).count()  # type: ignore
+            col_indexes: List[Dict[str, Any]] = self.db.collection(col_name).indexes()  # type: ignore
+
+            indexes = []
+            for index in col_indexes:
+                # Add required fields
+                index_dict = {
+                    "id": index["id"],
+                    "fields": index["fields"],
+                }
+                # Add optional fields only if they exist
+                for field in ["type", "name"]:
+                    if value := index.get(field):
+                        index_dict[field] = value
+                indexes.append(index_dict)
```

Comment:
Similar to the `schema_include_views` parameter (defaults to `False`), let's introduce a `schema_include_indexes` parameter, which if set to `True`, will generate the indexes and include it in the Graph Schema.

The motivation for this is to keep the next versions of `ArangoGraph` backwards compatible, with new features capable of being enabled via a Boolean parameter